### PR TITLE
ENH: Update builds for ITK 5.2 RC 3 ++

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

**Review by dzenanz — APPROVED** [review]

File: .github/workflows/build-test-package.yml [hunk 1]
```diff
@@ -13,17 +13,17 @@ jobs:
           - os: ubuntu-18.04
             c-compiler: "gcc"
             cxx-compiler: "g++"
-            itk-git-tag: "v5.2rc02"
+            itk-git-tag: "c7586a12e8538c59041c480880ff0ea15c9550c2"
```
Comment (N-Dekker) [inline .github/workflows/build-test-package.yml:16]: The subject of the commit message says "Update builds for ITK 5.2 RC 3", so didn't you intend to specify `itk-git-tag: "v5.2rc03"`?

Comment (N-Dekker) [reply .github/workflows/build-test-package.yml:16]: Or possibly even `itk-git-tag: v5.2.0`?

Comment (thewtex) [reply .github/workflows/build-test-package.yml:16]: Windows CI has issues with the v5.2rc02 tag, so this is v5.2rc02++ :smile_cat: .

Comment (thewtex) [reply .github/workflows/build-test-package.yml:16]: @N-Dekker verifying that we have the opencl builds working with current elastix `develop`.

**Pull request merged**